frontend/src/components/StatusTag.vue

```diff
@@ -189,11 +189,11 @@ export default {
       return `statusTag_${this.popperKey}`
     },
     tag () {
-      const { lastTransitionTime, lastUpdateTime, message, status, type, codes } = this.condition
+      const { lastTransitionTime, message, status, type, codes } = this.condition
       const id = type
       const { displayName: name, shortName, description } = this.conditionMetadataFromType(type)
 
-      return { id, name, shortName, description, message, lastTransitionTime, lastUpdateTime, status, codes }
```

Comment:
This line should also be removed https://github.com/gardener/dashboard/blob/dbcd830eb6a41e527f10c0ac3ce41ad533dd6d33/frontend/src/components/StatusTag.vue#L61

Reply:
good catch, thanks!
done